apps/hexagon_matmul/Makefile

```diff
@@ -0,0 +1,55 @@
+include ../support/Makefile.inc
```

Comment:
I don't think HelloHexagon should be fixed in this PR, but I'd rather not commit new code that will immediately need fixing to match the structure of the other apps.